src/tabs/tabs.js

```diff
@@ -132,6 +133,10 @@ angular.module('ui.bootstrap.tabs', [])
         });
       }
 
+      if (attrs.classes === undefined) {
```

Comment:
Do I need to fix this, resquash, rebase, etc. or this is within the margin of tolerance?